app/test_main.py

```diff
@@ -1,3 +1,57 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_value",
+    [
+        pytest.param(
+            14, 14,
+            [0, 0],
+            id="should return zero when dog-cat ages equal 0"
+        ),
+        pytest.param(
+            14, 14,
+            [0, 0],
+            id="should return zero when dog-cat ages less 14"
+        ),
```

Comment:
The test cases on line 9 and line 14 are identical, which contradicts the 'Don't Repeat Yourself' principle from the checklist. Consider removing the duplicate.